src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let action of actions) {
+    switch (action.type) {
+      case ('clear'):
```

Comment:
```suggestion
      case 'clear':
```
you can remove brackets